cosmic_ray/modules.py

```diff
@@ -26,7 +26,7 @@ def fixup_module_name(name):
     return name
 
 
-def find_modules(name, excludes=None):
+def find_modules(_name, excludes=None):
```

Comment:
Why did you make this change? I see that there's a duplication of `name` below, but you seem to have renamed them both.